README.md

```diff
@@ -17,18 +17,18 @@ $ brew install proj freetds
 $ asdf install
 $ asdf reshim
 $ pip install pipenv
-$ pipenv install  # --dev if you want to do development on the scripts
+$ pipenv sync  # --dev if you want to do development on the scripts
 $ cp .env.template .env
 $ vi .env  # configure environment variable
 ```
 
 ### Windows
 ```
 $ choco install pyenv-win
-$ pyenv install
+$ pyenv install 3.9.2
```

Comment:
This gives an error: 
 pyenv-install: definition not found: 3.9.2

The latest I see is 3.9.0a4 .

Reply:
That sounds like your pyenv is out of date: try `pyenv update`?